src/main/java/com/exactpro/epfast/template/xml/DictionaryFieldOperatorXml.java

```diff
@@ -16,32 +16,28 @@
 
 package com.exactpro.epfast.template.xml;
 
-import com.exactpro.epfast.template.Dictionary;
-
-import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlAttribute;
 
 public class DictionaryFieldOperatorXml extends FieldOperatorXml {
 
-    private Dictionary dictionary;
-
-    private ReferenceImpl dictionaryKey;
+    private ApplicationIdReferenceImpl dictionaryKey = new ApplicationIdReferenceImpl(this);
 
-    public Dictionary getDictionary() {
-        return dictionary;
+    @XmlAttribute(name = "dictionary")
+    public void setDictionaryName(String dictionary) {
+        super.setDictionaryName(dictionary);
     }
 
-    @XmlElement(name = "dictionary", namespace = NamespaceProvider.XML_NAMESPACE)
-    public void setDictionary(Dictionary dictionary) {
-        this.dictionary = dictionary;
+    public ApplicationIdReferenceImpl getDictionaryKey() {
+        return dictionaryKey;
     }
 
-    public ReferenceImpl getDictionaryKey() {
-        return dictionaryKey;
+    @XmlAttribute(name = "keyName")
+    public void setDictionaryKeyName(String keyName) {
+        dictionaryKey.setName(keyName);
     }
 
-    @XmlElement(name = "nsKey", namespace = NamespaceProvider.XML_NAMESPACE)
-    public void setDictionaryKey(ReferenceImpl dictionaryKey) {
-        this.dictionaryKey = dictionaryKey;
+    @XmlAttribute(name = "keyNs")
```

Comment:
Is it "nsKey" ? I think it's just "ns" and it should be delegated to `super.setApplicationNamespace()`. Please double-check that.